generator/generic_generator_utilities.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+module Inferno
+  module Generator
+    module GenericGeneratorUtilties
```

Comment:
I don't think this module forms a cohesive unit.

- `structure_to_string` is only used by `GenericGenerator`
- `create_search_validation` is only used by `SequenceMetadata`
- the other methods are helpers for `create_search_validation`

The name doesn't help you know why a method would be in here rather than in one of those classes.

Reply:
I wanted to leave the `create_search_validation` stuff out of `SequenceMetadata` because I wanted to keep that file for just profile information.  `create_search_validation` just uses the metadata to form code and doesn't add any metadata. I couldn't put it in `search_test` either because it doesn't go inside of a test itself. 

`structure_to_string` is going to be needed later on when I add must support stuff.